packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -329,9 +329,40 @@ const VirtualListBaseFactory = (type) => {
 
 			if (this.props.itemSizes) {
 				if (this.itemPositions.length > this.props.itemSizes.length) {
+					// The item with `this.props.itemSizes.length` index is not rendered yet.
+					// So the item could scroll into view after it rendered.
+					// To do it, `this.props.itemSizes.length` value is cached in `this.indexToScrollIntoView`.
+					this.indexToScrollIntoView = this.props.itemSizes.length;
```

Comment:
I need some explanation why the last index of `itemSizes` array should be the index to show. And, `indexToScrollIntoView` will be the last index + 1 with this code if I read understand correctly.